test/sanitize-field-names/hapi.test.js

```diff
@@ -64,7 +64,7 @@ async function runTest(
   agent._config(agentConfig);
   const server = Hapi.server({
     port: 0,
-    host: 'localhost',
+    host: '127.0.0.1',
```

Comment:
Reviewer note: The test env inside the chainguard container was hitting the failing case we've seen before where (a) the test server thinks "localhost" is IPv6 and (b) the test client uses IPv4 (127.0.0.1). This fixes that by dropping the ambiguity.